src/main/java/uk/gov/hmcts/ccd/domain/service/message/CaseEventMessageService.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.ccd.domain.service.message;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ccd.data.casedetails.CaseAuditEventRepository;
+import uk.gov.hmcts.ccd.data.message.MessageCandidateRepository;
+import uk.gov.hmcts.ccd.data.user.CachedUserRepository;
+import uk.gov.hmcts.ccd.data.user.UserRepository;
+import uk.gov.hmcts.ccd.domain.model.std.MessageInformation;
+import uk.gov.hmcts.ccd.domain.model.std.MessageQueueCandidate;
+
+import javax.inject.Inject;
+import java.time.Clock;
+
+@Service
+@Qualifier("caseEventMessageService")
+public class CaseEventMessageService extends AbstractMessageService {
+
+    private static final ObjectMapper mapper = new ObjectMapper();
```

Comment:
if it's not a time consuming change (considering having to refactor tests) can you inject the mapper picking one already configured in `JacksonObjectMapperConfig`?

Reply:
done 👍